src/bin/release-please.ts

```diff
@@ -125,13 +126,18 @@ export const parser = yargs
   .option('token', {describe: 'GitHub token with repo write permissions'})
   .option('api-url', {
     describe: 'URL to use when making API requests',
-    default: 'https://api.github.com',
+    default: GH_API_URL,
     type: 'string',
   })
   .option('default-branch', {
     describe: 'The branch to open release PRs against and tag releases on',
     type: 'string',
   })
+  .option('fork', {
+    describe: 'should the PR be created from a fork',
+    type: 'boolean',
```

Comment:
Reading through `factory.ts`, It didn't look like `fork` is passed to `githubInstance` in the `releasePR` helper?

Does it still need to be set here so that the branch can be created from a fork?

Reply:
good catch, that means I'm missing a test! will fix (and I think there's an even simpler way to pass around Factory config options to the different factory helpers)